### PR TITLE
Adding Bulgarian cg and updating Icelandic cg in the repo

### PR DESCRIPTION
**Pull request by Othergreengrasses**
Adding Bulgarian covering grammar in the repo. Icelandic covering grammar has some Unicode issue. Uploading updated file.

### CODE REVIEW

File: data/covering_grammar/tsv/ice_latn_broad.tsv [hunk 1]
```diff
@@ -1,140 +1,123 @@
...
+v	v
+x	s
+x	ks
+x	xs
+y	i
+y	ɪ
+y	ɪː
+ý	i
+ý	iː
+
```
Comment (kylebgorman) [inline data/covering_grammar/tsv/ice_latn_broad.tsv:122]: I don't know for sure but I think two blank lines is different than (just) one.

**Review by kylebgorman — APPROVED** [review]
LGTM but I didn't look closely. Feel free to assign another team member if you'd like a closer look.

File: data/covering_grammar/tsv/ice_latn_broad.tsv (hunk 1 above)
Comment (Othergreengrasses) [reply data/covering_grammar/tsv/ice_latn_broad.tsv:122]: Hmm, interesting. It's not making any difference in the result though.

**Comment by Othergreengrasses** [comment]
Please merge this PR if it's LGTY.

**Pull request merged**